example/mirroring/web/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Css mirroring Example</title>
+    <script async src="main.dart" type="application/dart"></script>
+    <script async src="packages/browser/dart.js"></script>
+</head>
+<body>
+<foo></foo>
+<div style="text-align:center">
```

Comment:
please use a class (`sidebar` would be readable, for instead) instead of inline style